test1.txt

```diff
@@ -0,0 +1,16 @@
+Just some ipsm lorem stuff
+for a few lines to
+make a change
+big enough to
+surpass diff
+basic line
```

Comment:
[On the Git mailing list](https://github.com/webstech/gggmaildf2deeb7-4e50-d90e-1720-bb8f8ffe8967@example.com), Code Bot wrote ([reply to this](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis)):

``````````email
This email is being reviewed.

   From: "Chris. Webster" <chris@webstech.net>
   
   2022-12-25 15:59:14.10
   
   test1.txt
   
   Signed-off-by: Chris. Webster <chris@webstech.net>
   ---
    test1.txt | 8 ++++++++
    1 file changed, 8 insertions(+)
   
   diff --git a/test1.txt b/test1.txt
   index 5ade0ce..750fdba 100644
   --- a/test1.txt
   +++ b/test1.txt
   @@ -6,3 +6,11 @@ surpass diff
    basic line
    limits.
    15:59:08.69  
   +Just some ipsm lorem stuff
   +for a few lines to
   +make a change
   +big enough to
   +surpass diff
   +basic line
   +limits.
   +15:59:14.10  
   -- 
   webstech
   
   
``````````

Reply:
Line number is correct here.

Reply:
Why does this show up in commit 1 on the commit 1 review?

Reply:
The diff is incorrect showing + signs when there should not be a plus sign.